src/backend/common/datafeeds/parsers/fms_api/fms_api_event_alliances_parser.py

```diff
@@ -0,0 +1,59 @@
+from typing import Any, cast, Dict, List, Optional
+
+from backend.common.datafeeds.parsers.parser_json import ParserJSON
+from backend.common.models.alliance import EventAlliance, EventAllianceBackup
+
+
+class FMSAPIEventAlliancesParser(ParserJSON[Optional[List[EventAlliance]]]):
+    def parse(self, response: Dict[str, Any]) -> Optional[List[EventAlliance]]:
```

Comment:
Note to Zach: Either this should return no event alliances, or `FMSAPIDistrictListParser` should return an `Optional`. I'm leaning towards the latter, but you'll need to be careful on the integration component/what it means to have no data.